### PR TITLE
Removed inbox dividers

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

**Comment by cecileboucheron** [comment]
@adamjspooner mind taking a look?

**Comment by adamjspooner** [comment]
Taking a look now.

File: shared/chat/inbox/row/big-team-header/index.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ const styles = styleSheetCreate({
   },
   team: platformStyles({
     common: {
+      color: globalColors.black_60,
```
Comment (adamjspooner) [inline shared/chat/inbox/row/big-team-header/index.js]: I prefer `black_75`, but it's not a deal breaker.

Comment (cecileboucheron) [reply shared/chat/inbox/row/big-team-header/index.js]: I like it. `black_60` is BodySmallSemiboldSecondaryLink and consistent with our clickable section labels.

**Pull request merged**